examples/app_p2p_DTR/cload-all.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
This is a pretty generic swarm flashing script. Maybe it should be generalized and documented elsewhere?

Reply:
It was moved to tools/utils